single-programacion.php

```diff
@@ -103,6 +104,30 @@
                 ?>
                 <a href="https://www.facebook.com/sharer/sharer.php?u=<?php the_permalink(); ?>" target="_blank" rel="noopener noreferrer"><?php _e('[:en]Share on Facebook[:es]Compartir en Facebook'); ?> ></a>
               </div>
+              <?php 
+               if (!empty($actividades)) {
+              ?> 
+              <div class="col col-s-12 col-m-6 col-l-4">
+                <h2 class="margin-bottom-tiny"><?php echo __('[:es]Actividad Académica[:en]Academic Activity'); ?></h2>
+                <ul class="font-serif">
+              <?php 
+                foreach ($actividades as $actividad) {
+                  $activity_num = '';
+
+                  if (!empty($number)) {
+                    $activity_num = 'No. ' . add_leading_zero($number) . '.' . get_actividad_num($actividad->ID);
+                  }
+              ?> 
+                  <li class="margin-bottom-tiny"><a href="<?php echo get_permalink($actividad->ID); ?>"><?php 
```

Comment:
![screen shot 2016-11-01 at 6 03 32 pm](https://cloud.githubusercontent.com/assets/7727722/19912074/8addc42e-a05d-11e6-9e02-139218feee05.png)

it does tho